aten/src/ATen/core/ivalue_inl.h

```diff
@@ -307,6 +310,15 @@ struct C10_EXPORT ivalue::Future final : c10::intrusive_ptr_target {
     return value_;
   }
 
+  const IValue& constValue() {
```

Comment:
This is not needed right? we only need to use it in the generic future, and the const reference of an IValue seems obsolete.

Reply:
It's needed.
Notice this PR is to migrate usages of generic future to ivalue future. 
In "rref_impl.cpp", `OwnerRRef::getValue()` uses `future_->contValue()`.